sys/arm64/arm64/support.S

```diff
@@ -218,6 +238,26 @@ EENTRY(suword64)
 EEND(suword64)
 END(suword)
 
+#if __has_feature(capabilities)
+/*
+ * int sucap(volatile void * __capability, intcap_t)
+ */
+ENTRY(sucap)
+	ldr	x2, =(VM_MAXUSER_ADDRESS-7)
+	cmp	x0, x2
+	b.cs	fsu_fault_nopcb
+	adr	x6, fsu_fault		/* Load the fault handler */
+	SET_FAULT_HANDLER(x6, x2)	/* And set it */
+	ENTER_USER_ACCESS(w6, x4)
+	/* TODO: Store through the capability */
```

Comment:
If this is not a complete implementation it should not be committed.

Reply:
There was an issue where we were passed an untagged capability. I haven't had time to track it down.